packages/sdk-components-react-radix/src/radix-collapsible.ws.ts

```diff
@@ -0,0 +1,106 @@
+import { RadioUncheckedIcon, RadioCheckedIcon } from "@webstudio-is/icons/svg";
+import type {
+  WsComponentMeta,
+  WsComponentPropsMeta,
+} from "@webstudio-is/react-sdk";
+import {
+  propsRadixCollapsible,
+  propsRadixCollapsibleContent,
+  propsRadixCollapsibleTrigger,
+} from "./__generated__/radix-collapsible.props";
+
+export const metaRadixCollapsible: WsComponentMeta = {
+  category: "radix",
+  type: "container",
+  label: "Collapsible",
+  icon: RadioUncheckedIcon,
+  template: [
+    {
+      type: "instance",
+      component: "RadixCollapsible",
+      props: [
+        {
+          name: "open",
+          type: "boolean",
+          value: false,
```

Comment:
should open be false by default? My initial reaction when testing: "where is content?"

Reply:
Dialogs and tooltips are closed by default too